pkg/code-generator/codegen/templates/snapshot_emitter_template.go

```diff
@@ -260,13 +284,32 @@ func (c *{{ lower_camel .GoName }}Emitter) Snapshots(watchNamespaces []string, o
 {{- if .ClusterScoped }}
 			case {{ lower_camel .Name }}List := <- {{ lower_camel .Name }}Chan:
 				record()
+
+				stats.RecordWithTags(
+					ctx,
+					[]tag.Mutator{
+						tag.Insert({{ lower_camel $resource_group }}NamespaceKey, "cluster-scoped"),
```

Comment:
i think that in gloo i use `<all>` for cluster scoped namespaces.
I like `<all>` because it is not a valid namespace name.

see here: https://github.com/solo-io/solo-kit/blob/97135f06fc5332dc6a619106e007f89e0518f6aa/pkg/api/v1/clients/kube/resource_client_factory.go#L120

I'm open to change this as long as we are consistent.